web/src/server/api/services/tableDefinitions.ts

```diff
@@ -273,4 +273,16 @@ export const tableDefinitions: TableDefinitions = {
       traceTags,
     ],
   },
+
+  dataset_items: {
+    table: ` dataset_items di`,
```

Comment:
same here, which UI tables require this?

Reply:
Same as above, will move to different file to make clearer this is not related to tables